From 227f28843e33d7531294112daca4c9b81e307ad6 Mon Sep 17 00:00:00 2001
From: Michal Vasilek <michal.vasilek@nic.cz>
Date: Fri, 16 Jun 2023 18:00:29 +0200
Subject: [PATCH] updater-supervisor: fix build with python3 -m installer

on OpenWrt 23.05+, updater-supervisor is built with pep517 which
generates a wheel with name svupdater, but the build system expects
the name updater-supervisor by default
---
 updater/updater-supervisor/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/updater/updater-supervisor/Makefile b/updater/updater-supervisor/Makefile
index 0825350d1..b6b0f020b 100644
--- a/updater/updater-supervisor/Makefile
+++ b/updater/updater-supervisor/Makefile
@@ -22,6 +22,8 @@ PKG_LICENSE:=MIT
 include $(INCLUDE_DIR)/package.mk
 include $(TOPDIR)/feeds/packages/lang/python/python3-package.mk
 
+PYTHON3_PKG_WHEEL_NAME:=svupdater
+
 define Package/updater-supervisor
   SECTION:=updater
   CATEGORY:=Turris Updater
-- 
2.41.0

